modules/apps/calendar/calendar-web-test/src/testFunctional/tests/AddCalendarAccessibility.testcase

```diff
@@ -76,4 +76,35 @@ definition {
 		}
 	}
 
+	@description = "LPS-185089 Verifies that the focus is not lost after closing the 'Edit Event' modal."
+	@priority = 3
+	test FocusIsNotLostAfterClosingTheModal {
+		task ("And a new event is added") {
+			CalendarNavigator.gotoAddEvent();
+		}
+
+		task ("When the user tries to edit the event") {
+			CalendarEvent.editTitle(eventTitle = "Calendar Event Title");
+		}
+
+		task ("And clicks any 'Select' button on 'CATEGORIZATION'") {
+			CalendarEvent._expandPanel(panelName = "Categorization");
+
+			ScrollWebElementIntoView(locator1 = "CalendarEditEvent#RELATED_ASSETS_PANEL_COLLAPSED");
```

Comment:
Hey @john-co, I tried scrolling to `Button#SELECT_CATEGORIES` but it does not work as expected, the button is not visible, therefore, it is not focusable... I think the way it is now is our best alternative.